docs/send-data/opentelemetry-collector/data-source-configurations/additional-configurations-reference.md

```diff
@@ -13,7 +13,7 @@ Sumo Logic Distribution for OpenTelemetry stores the configuration for the colle
 | :-- | :-- |
 | Linux | `/etc/otelcol-sumo/conf.d` |
 | Mac | `/etc/otelcol-sumo/conf.d` |
-| Windows | `C:\ProgramData\Sumo Logic OpenTelemetry Collector\` |
+| Windows | `C:\ProgramData\Sumo Logic OpenTelemetry Collector\` | 
```

Comment:
[@sumoanema](https://github.com/sumoanema) [@arunpatyal](https://github.com/arunpatyal) [@ajaiswals](https://github.com/ajaiswals) , can you confirm the Windows path mentioned above is correct? cc @angadrandhawa1